var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -35,10 +37,6 @@ class PyPytest(PythonPackage):
 
     # python_requires
     depends_on("python@3.7:", when="@7.1:", type=("build", "run"))
```

Comment:
7.3.2 is also the first version to add Python 3.12 support if you want to code that in: https://github.com/pytest-dev/pytest/releases/tag/7.3.2

Reply:
How would I do this? Isn't that already included by `python@3.7:`

Reply:
I'm thinking something like:
```python
depends_on("python@:3.11", when="@:7.3.1", type=("build", "run"))
```

Reply:
Ah, ok, thanks. Now I understand what you mean.
Done.